src/main/java/racingcar/model/Car.java

```diff
@@ -12,8 +12,8 @@ public Car(String name, int startPosition) {
         this.position = startPosition;
     }
 
-    public boolean canMoving(int randomValue) {
-        if (randomValue >= MIN_MOVING_NUM) {
+    public boolean canMoving(int power) {
+        if (power >= MIN_MOVING_NUM) {
             return true;
         }
         return false;
```

Comment:
if문을 통해 단순히 true, false를 반환하는 경우 이렇게 할수도있다는걸 말씀드린거였습니다 ㅎㅎ IDE에서 주의사항을 확인해보고 이유를 찾아보는것도 좋은 학습 방법이라고 생각해요!

```suggestion
    public boolean canMoving(int power) {
        return power >= MIN_MOVING_NUM;
```

<img width="384" alt="image" src="https://user-images.githubusercontent.com/39271364/218243609-c7e169cf-3e79-491c-87ca-6dfb372c14e3.png">

Reply:
감사합니다! 수정하였습니다!